edenai_apis/llmengine/utils/moderation.py

```
@@ -46,6 +46,24 @@ async def moderate_content(headers, content: Union[str, List]) -> bool:
+    api_key = api_settings.get("api_key")
+    headers = {"Authorization": f"Bearer {api_key}"}
+
+    tasks = []
+
+    messages = kwargs.get("messages", [])
+
+    for message in messages:
+        if "content" in message:
+            tasks.append(moderate_content(headers, message["content"]))
+
+    async with aiohttp.ClientSession() as session:
```

Ruff (0.8.2)

Local variable `session` is assigned to but never used

Remove assignment to unused variable `session`

(F841)